### ISSUE TITLE
Improved install instructions for Mac

### ISSUE THREAD

**Issue by menelic** [ccbogel/QualCoder#280]
The install instructions for Mac are incomplete and hard to follow for novice users. They require additional tinkering and are therefore not as helpful as they could be esp. for users without advanced knowledge of the command line, python packages etc. Here is a more complete how-to that could be incorporated into the install instructions in the README, in the manual and on the website. This has led to successful installs on systems where the install instructions from the README did not lead to a successful install

 ---> install VLC, Python3 
 --> downlad Qualcoder-master Zip file and copy it into /Applications 

 --> in a Terminal

 curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
python3 --version
 ----> select  3.9 as an answer
python3 get-pip.py
cd /usr/local/bin
sudo ln -s ../../../Library/Frameworks/Python.framework/Versions/3.9/bin/pip pip
pip install pyserial
pip install pyqt5 lxml pillow six ebooklib ply chardet pdfminer.six openpyxl

/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
brew install qpdf

cd "/Applications/QualCoder-master/qualcoder"
python3 qualcoder.py

**Comment by ccbogel** [comment]
Thank you I have added this to the read me.
I will update the manual.

**Comment by ccbogel** [comment]
@menelic thanks I have updated the instructions
